src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,99 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0a3 (9/11/2020)
+------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Add `runtime_binary_dependencies` field to `python_tests` target (#10756)
+  `PR #10756 <https://github.com/pantsbuild/pants/pull/10756>`_
+  `PR #10744 <https://github.com/pantsbuild/pants/pull/10744>`_
+
+* Add new intrinsic `Paths` to avoid snapshotting (#10741)
+  `Issue #10711.
 <https://github.com/pantsbuild/pants/issues/10711.
>`_
+  `PR #10741 <https://github.com/pantsbuild/pants/pull/10741>`_
+
+* Replace `cloc` with `count-loc`, using much faster SCC program (#10740)
+  `PR #10740 <https://github.com/pantsbuild/pants/pull/10740>`_
+
+* Upgrade default isort from 4.6 to 5.5 (#10737)
+  `Issue #10727.
 <https://github.com/pantsbuild/pants/issues/10727.
>`_
+  `PR #10737 <https://github.com/pantsbuild/pants/pull/10737>`_
+  `PR #10460 <https://github.com/pantsbuild/pants/pull/10460>`_
+
+* Disambiguate binary and lambda output. (#10735)
+  `PR #10735 <https://github.com/pantsbuild/pants/pull/10735>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Pass proxy-related env vars through to Pex calls. (#10762)
+  `PR #10762 <https://github.com/pantsbuild/pants/pull/10762>`_
+
+* Support custom CA bundles. (#10757)
+  `PR #10757 <https://github.com/pantsbuild/pants/pull/10757>`_
+
+* Add support to MyPy for first-party plugins (#10755)
+  `PR #10755 <https://github.com/pantsbuild/pants/pull/10755>`_
+
+* Add support to MyPy for third-party requirements and stubs (PEP 561) (#10753)
+  `PR #10753 <https://github.com/pantsbuild/pants/pull/10753>`_
+
+* Support binary dependencies in tests. (#10744)
+  `PR #10744 <https://github.com/pantsbuild/pants/pull/10744>`_
+
```

Comment:
Can move to refactoring. `Add `runtime_binary_dependencies` field to `python_tests` target` changes the implementation, and we can have that be the "new feature".

This is "prework"